doc/nrf/libraries/dfu/suit_dfu.rst

```diff
@@ -0,0 +1,50 @@
+.. _subsys_suit:
+
+Software Updates for Internet of Things (SUIT)
+##############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Software Updates for Internet of Things (SUIT) DFU library provides functionality for a Local domain Core to orchestrate the update based on the SUIT manifest.
+
+Overview
+********
+
+The SUIT DFU library provides APIs to start as well as prepare the device before the update procedure is started.
+The preparation for the DFU may be described as a sequence inside a SUIT manifest (suit-payload-fetch).
+To interpret and execute this sequence, a SUIT processor must be instantiated inside the Local domain Core firmware.
+This library provides a simplified API, that manages the processor state, executes the fetching sequence from the SUIT manifest, as well as passes the update candidate info to the Secure domain.
+
+The following steps are required by the application to perform the Device Firmware Update:
+
+1. Call :c:func:`suit_dfu_initialize` to initialize the SUIT DFU module before any other SUIT functionalities are used.
+   This function has to be called only once. This function must be called with at least APPLICATION init level
+   (calling it in POST_KERNEL or earlier init level causes undefined behavior).
```

Comment:
```suggestion
   
   
```suggestion
   Calling this function in ``POST_KERNEL`` or earlier init level causes undefined behavior.
```